### ISSUE TITLE
Parallel upload of multipart file

### ISSUE THREAD

**Issue by imonarock** [MoriTanosuke/glacieruploader#59]
Is it possible to upload the parts in parallel with your uploader?

**Comment by MoriTanosuke** [comment]
No, the upload is single threaded at the moment. Multi thread would require to rewrite this part:

https://github.com/MoriTanosuke/glacieruploader/blob/6889a4404e4c29d7011d15593a9345a1792eb2e2/glacieruploader-command/src/main/java/de/kopis/glacier/commands/UploadMultipartArchiveCommand.java#L143-L166
